Compact sidebar

isc-etamarch opened the pull request:
Fixes #525

isc-etamarch committed:
Make the sidebar more compact

Compacted the sidebar;
Fixed alignment of icons

isc-etamarch committed:
Add bottom border to top sidebar section

isc-etamarch commented:
@isc-tleavitt Let me know if we need further compacting / other UI changes to the sidebar

isc-tleavitt committed:
Merge branch 'main' into issue-525

isc-tleavitt reviewed (changes requested):
@isc-etamarch this is a vast improvement, thank you.

My main remaining concern is the behavior with highly limited vertical space - there might not be enough for the remote branch view. This is great:
![image](https://github.com/user-attachments/assets/39c4561d-4504-4e10-8d51-fb0257d41f54)

But this is not - from in VSCode (which is particularly egregiously bad for some reason - maybe there are other steps we could take re: zoom/etc. to make this better):
![image](https://github.com/user-attachments/assets/bd1f3758-0b31-48b3-8e34-93788391d02a)

Note the overlap of "Remote Branches" with the links underneath, and the fact that you cant see "More..."

I don't want to go too crazy on this, but let's try to make it a bit better first.

isc-tleavitt committed:
Merge branch 'main' into issue-525

isc-etamarch committed:
Fix bottom navigation

Added margins to the bottom navigation so that the remote branches would not be visible underneath. Also added padding below remote branches to that they can all be visible with scrolling

isc-etamarch committed:
Merge branch 'main' into issue-525
